site/app/models/Gradeable.php

```diff
@@ -331,6 +320,8 @@ public function __construct(Core $core, $details, User $user = null) {
                         if ($details['array_gcd_gc_id'][$j] === $component_details['gc_id']) {
                             $component_details['gcd_score'] = $details['array_gcd_score'][$j];
                             $component_details['gcd_component_comment'] = $details['array_gcd_component_comment'][$j];
+                            $component_details['gcd_grader_id'] = $details['array_gcd_grader_id'][$j];
```

Comment:
looking at it again, we can get rid of that line, but not the parts in the query because the creation of the grader User still relies on that field